CMakeLists.txt

```diff
@@ -204,7 +204,7 @@ include_directories(SYSTEM
   ${OPENGL_INCLUDE_DIR}
   ${PYTHON_INCLUDE_PATH}
 )
-include_directories(src ${catkin_INCLUDE_DIRS})
+include_directories(src SYSTEM ${catkin_INCLUDE_DIRS})
```

Comment:
Yeah, I was thinking that too. On the one hand it seems a bit overkill, but on the other hand a separate wrapper header would be the best way anyway if the offending include is used multiple times. And then `#pragma GCC system_header` is viable option again.

I'll go for that I suppose =]